iocore/net/P_SSLConfig.h

```diff
@@ -161,6 +159,25 @@ struct SSLCertificateConfig {
   static int configid;
 };
 
+struct SSLTicketParams : public ConfigInfo {
+  mutable ssl_ticket_key_block *default_global_keyblock;
```

Comment:
This should not be `mutable`. If you need that, you're doing something wrong elsewhere.